lib/index.js

```diff
@@ -144,35 +144,35 @@ Server.prototype = {
           try {
             this.processManager.spawn(proc); 
           } catch (error) {
-            bootstrapLogger.warn(`Could not spawn ${JSON.stringify(proc)}: ${error.message}`);
+            bootstrapLogger.warn(`ZWED0020W`, JSON.stringify(proc), error.message); //bootstrapLogger.warn(`Could not spawn ${JSON.stringify(proc)}: ${error.message}`);
           }  
         } else {
-          bootstrapLogger.log(bootstrapLogger.INFO, `Skip child process spawning on worker ${process.clusterManager.getIndexInCluster()} ${proc.path}\n`);
+          bootstrapLogger.info("ZWED0115I", process.clusterManager.getIndexInCluster(), proc.path); //bootstrapLogger.log(bootstrapLogger.INFO, `Skip child process spawning on worker ${process.clusterManager.getIndexInCluster()} ${proc.path}\n`);
         }
       }
     }
     const wsConfig = this.userConfig.node;
     if (!(yield this.webServer.validateAndPreprocessConfig(wsConfig))) {
       const httpsConfig = wsConfig.https;
       const httpConfig = wsConfig.http;
-      bootstrapLogger.warn('Missing one or more parameters required to run.');
-      bootstrapLogger.warn('The server requires either HTTP or HTTPS.'
-        + ' HTTP Port given: ' + (httpConfig? httpConfig.port : null)
-        + '. HTTPS Port given: ' + (httpsConfig? httpsConfig.port : null));
-      bootstrapLogger.warn('HTTPS requires either a PFX file or Key & Certificate files.');
-      bootstrapLogger.warn('Given PFX: '+ (httpsConfig? httpsConfig.pfx : null));
-      bootstrapLogger.warn('Given Key: '+ (httpsConfig? httpsConfig.keys : null));
-      bootstrapLogger.warn('Given Certificate: '+ (httpsConfig?
-          httpsConfig.certificates : null));
+      bootstrapLogger.warn('ZWED0021W'); //bootstrapLogger.warn('Missing one or more parameters required to run.');
```

Comment:
This doesn't translate well into message ID format, there are multiple here with the same meaning.
Since these all print at the same time why not just have 1 message ID with a bunch of newlines?

Reply:
Consolidated into a single message.